### PR TITLE
Add support for the new Link Field

### LINKED ISSUE #78: Add support for Link Field introduced in ACF 5.6
ACF 5.6 introduced the [Link Field](https://www.advancedcustomfields.com/resources/link/).

The handling should probably be pretty similar to the URL Field by just building an `<a>`.

### PR DESCRIPTION
**Pull request by bullpillaren**
As suggested in [#78](https://github.com/Yoast/yoast-acf-analysis/issues/78) - fixes #78

### CODE REVIEW

File: js/src/scraper-store.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ var scraperObjects = {
     'textarea':     require( "./scraper/scraper.textarea.js" ),
     'email':        require( "./scraper/scraper.email.js" ),
     'url':          require( "./scraper/scraper.url.js" ),
+    'link':          require( "./scraper/scraper.link.js" ),
```
Comment (andizer) [inline js/src/scraper-store.js]: Can you remove a single space before `require` to have it aligned with the other requires

File: js/src/scraper/scraper.link.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+var scrapers = require( "./../scraper-store.js" );
+
+var Scraper = function() {};
+
+Scraper.prototype.scrape = function(fields){
```
Comment (andizer) [inline js/src/scraper/scraper.link.js]: I am missing the documentation for the function.

File: js/src/scraper/scraper.link.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+    var that = this;
+
+    fields = _.map(fields, function(field){
```
Comment (andizer) [inline js/src/scraper/scraper.link.js]: Can you add some documentation for the callback function?

Comment (andizer) [inline js/src/scraper/scraper.link.js]: You might consider returning this directly like `return _.map(fields, function(field){`

File: js/src/scraper/scraper.link.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+var scrapers = require( "./../scraper-store.js" );
+
+var Scraper = function() {};
+
+Scraper.prototype.scrape = function(fields){
+
+    var that = this;
```
Comment (andizer) [inline js/src/scraper/scraper.link.js]: I looks like this variable isn't used in this method. You might consider removing it.

**Comment by kraftner** [comment]
Most oft the issues @andizer pointed out are totally valid but probably due to the fact that I proposed just copying the URL Field which has the same issues. :) Worth fixing anyway. I've also opened #100 to address this on a global scale.

Another thing I'd like to add is that this PR is lacking tests.

**Comment by bullpillaren** [comment]
Added some documentation, removed some unused variables and  added some tests for the link field. But It fails on 3 field types (`Image Field`, `Link Field`, `Repeater Field` now, I have no idea why. 

They all fail for the same reason, `Timed out while waiting for element <body.post-new-php> to be present for 15000 milliseconds.  - expected "visible" but got: "not found"`

Got an idea @kraftner?

**Comment by kraftner** [comment]
I just ran the tests on the PR and for me everything is working.

The error you are seeing happens when login into WP failed for whatever reason. You should see screenshots of the failed state in `tests/js/system/screenshots/*`. Maybe this hints at the problem.

**Comment by bullpillaren** [comment]
That's weird, all screenshots from that kind of error are broken, other screenshots are working. I'll try to find some time to debug this later today, but it looks like it's a problem with my setup.

**Comment by kraftner** [comment]
Sigh. That is the downside of those tests, this hard to debug errors...

I think I have seen that from time to time as well without any clear reason. What you can try is increasing the memory for the VM (see https://varyingvagrantvagrants.org/docs/en-US/vvv-config/#vm_config)

```
vm_config:
  memory: 2048
```

which has apparently made things more stable here.

**Comment by moorscode** [comment]
I have resolved the merge conflicts and made sure the JavaScript conforms to the standards introduced in `develop`.

**Pull request merged**